enterprise/frontend/src/embedding-sdk/cli/utils/get-package-version.ts

```diff
@@ -0,0 +1,38 @@
+import fs from "fs/promises";
+
+import path from "path";
+
+export const hasPackageJson = async () => {
+  try {
+    await fs.access("package.json");
+    return true;
+  } catch (error) {
+    return false;
+  }
+};
+
+export const getPackageVersion = async (packageName: string) => {
```

Comment:
this goes into the `node_modules` folder and finds the version of the current package, then we can display it in the CLI. The next step would be to check the version against the latest version and have a banner saying "upgrade now!"